### ISSUE TITLE
Add multi root editor description to predefined builds section

### ISSUE THREAD

**Issue by scofalik** [ckeditor/ckeditor5#13536]
## 📝 Provide a description of requested docs changes

The multi-root editor is missing from the [predefined builds](https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/predefined-builds.html) guide and should be added. This was extracted as a follow-up issue from [the main issue](https://github.com/ckeditor/ckeditor5/issues/11493).

**Comment by godai78** [comment]
Waiting, https://github.com/ckeditor/ckeditor5/pull/13532 must be merged first.
